### PR TITLE
removes redundancy from DepositAssets

### PR DESCRIPTION
**Pull request by Jenniferstrej**

### CODE REVIEW

**Comment by giorgiosironi** [comment]
I see nothing strange but I'll go through this on Monday, checking the unit test coverage.

**Comment by gnott** [comment]
I think the static method ``get_keys()`` might be not used anymore by this activity or any other and can probably be removed.

Otherwise looks clear and smart to exclude files from the full list of files and then copy those :+1:

File: activity/activity_DepositAssets.py [hunk 1]
```diff
@@ -32,73 +34,72 @@ def do_activity(self, data=None):
         version = session.get_value(run, 'version')
         article_id = session.get_value(run, 'article_id')
 
-        self.emit_monitor_event(self.settings, article_id, version, run, "Deposit assets", "start",
+        self.emit_monitor_event(self.settings, article_id, version, run, self.pretty_name, "start",
                                 "Depositing assets for " + article_id)
```
Comment (giorgiosironi) [inline activity/activity_DepositAssets.py]: The unit tests in `tests/activity/test_activity_deposit_assets.py` do not cover `do_activity`. I think two single positive and negative tests are necessary (one for the `return True` case and on for `ACTIVITY_PERMANENT_FAILURE`).

Comment (Jenniferstrej) [inline activity/activity_DepositAssets.py]: Tests have been added.

**Pull request merged**